libbeat/docs/newdashboards.asciidoc

```diff
@@ -31,182 +31,198 @@ The following topics provide more detail about importing and working with Beats
 
 You can use the `import_dashboards` script to import all the dashboards and the index pattern for a Beat, including the dependencies such as visualizations and searches.
 The `import_dashboards` script is available under
-https://github.com/elastic/beats/tree/master/libbeat/dashboards[beats/libbeat/dashboards], and it's copied in each Beat package under the `scripts` directory.
+https://github.com/elastic/beats/tree/master/libbeat/dashboards[beats/libbeat/dashboards]. It's also available in each Beat package under the `scripts` directory.
+
+There are a couple of common use cases for importing dashboards:
 
+* Users who are getting started with Beats may want to <<import-single-beat,import dashboards and/or the index pattern for a single Beat>>.
+* Community Beats developers may want to <<import-dashboards-for-development,import dashboards for development>> to use as a starting point for new dashboards.
+
+[[import-single-beat]]
+==== Import Dashboards and/or the Index Pattern for a Single Beat
 
 Using the `import_dashboards` script, you can import the dashboards and the index pattern to
 Elasticsearch running on localhost for a single Beat (eg. Metricbeat):
 
 - from a local directory:
-
++
 [source,shell]
 ----------------------------------------------------------------------
-$ ./scripts/import_dashboards -dir kibana/metricbeat
```

Comment:
@monicasarbu Yes. I removed it for a couple of reasons. We weren't using it consistently. Also, we've heard from customers (specifically, for cloud) that the $ makes it harder to copy/paste commands. Granted...we're not consistent in the doc, but we should move towards removing it. If we think that showing the $ adds value, then we should handle it as part of the formatting that happens in the doc build when we've tagged something as shell.